package.json

```diff
@@ -6,13 +6,15 @@
   "license": "GPL-2.0-or-later",
   "devDependencies": {
     "@babel/core": "^7.9.6",
+    "@babel/plugin-proposal-class-properties": "^7.10.1",
+    "@babel/plugin-proposal-private-methods": "^7.10.1",
     "@babel/plugin-transform-modules-amd": "^7.9.6",
     "@babel/polyfill": "^7.8.7",
     "@babel/preset-env": "^7.8.6",
     "autoprefixer": "^9.8.0",
+    "babel-eslint": "^11.0.0-beta.2",
     "babel-loader": "^8.0.6",
     "browser-sync": "^2.26.7",
-    "clean-webpack-plugin": "^3.0.0",
```

Comment:
Is there a reason for that? I'd consider it good to cleanup the results before the build (so there are no remaining files that don't belong there)

Reply:
That's handled by Gulp.